src/conf/Docker/Dockerfile-luw

```diff
@@ -1,3 +1,21 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Hi @garydgregory 

Not 100% sure if this is correct. I'm asking you cos was you who push the file.